### PR TITLE
Fix/types

### PR DESCRIPTION
**Pull request by Mehdi-Verfaillie**

### CODE REVIEW

File: src/types/websockets/request.ts [hunk 1]
```diff
@@ -198,7 +198,7 @@ export interface WsPrivateChannelArgTickers extends WsBaseRequestArg {
 }
 
 export interface WsPrivateChannelArgWithCcy extends WsBaseRequestArg {
-  channel: 'account' | 'account-greeks';
+  channel: 'account' | 'account-greeks' | 'withdrawal-info';
```
Comment (Mehdi-Verfaillie) [inline src/types/websockets/request.ts:201]: reference: https://www.okx.com/docs-v5/en/#websocket-api-private-channel-withdrawal-info-channel

**Comment by Mehdi-Verfaillie** [comment]
@tiagosiebler Ready for review

File: examples/ws-private.ts [hunk 2]
```diff
@@ -152,6 +152,12 @@ wsClient.subscribe({
   ccy: 'BTC',
 });
 
+// Withdrawal events for specific symbol only
+wsClient.subscribe({
+  channel: 'withdrawal-info',
+  ccy: 'BTC',
+});
+
```
Comment (tiagosiebler) [inline examples/ws-private.ts:160]: <3

**Review by tiagosiebler — APPROVED** [review]
LGTM

**Pull request merged**